src/test/java/com/thebois/models/world/WorldTests.java

```diff
@@ -291,4 +301,151 @@ public void getTerrainsReturnsTerrainEvenIfStructuresAreOnTopOfTerrain() {
         assertThat(actualNumberOfTerrainTiles).isEqualTo(expectedNumberOfTerrainTiles);
     }
 
+    private static Stream<Arguments> getCorrectCoordinatesToTest() {
```

Comment:
style

This name is in the correct format, but also quite unclear. What is "correct"? And it's obvious that they are used "ToTest", we're in a test-class.

`getCoordinatesOfPositionAndClosestStructure` would fit.